src/witan/models/dem/ccm/components_functions.clj

```diff
@@ -23,6 +25,33 @@
         (ds/rename-columns {col-to-avg avg-name})
         (wds/rollup :mean avg-name [:gss-code :sex :age]))))
 
+(defn jumpoffyr-method-trend
+  "Takes in a dataset with historical data, a column name for a trend to be 
```

Comment:
I'm confused by your use of the verb "plot". I thought it meant tracing on a graph. Is there another meaning?